tests/aws/services/sns/test_sns.snapshot.json

```diff
@@ -4181,5 +4181,619 @@
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "sub-attrs": {
+        "Attributes": {
+          "ConfirmationWasAuthenticated": "false",
+          "DeliveryPolicy": {
+            "healthyRetryPolicy": {
+              "minDelayTarget": 1,
+              "maxDelayTarget": 1,
+              "numRetries": 0,
+              "numMaxDelayRetries": 0,
+              "numNoDelayRetries": 0,
+              "numMinDelayRetries": 0,
+              "backoffFunction": "linear"
+            },
+            "sicklyRetryPolicy": null,
+            "throttlePolicy": {
+              "maxReceivesPerSecond": 1000
+            },
+            "requestPolicy": null,
+            "guaranteed": false
+          },
+          "EffectiveDeliveryPolicy": {
+            "healthyRetryPolicy": {
+              "minDelayTarget": 1,
+              "maxDelayTarget": 1,
+              "numRetries": 0,
+              "numMaxDelayRetries": 0,
+              "numNoDelayRetries": 0,
+              "numMinDelayRetries": 0,
+              "backoffFunction": "linear"
+            },
+            "sicklyRetryPolicy": null,
+            "throttlePolicy": {
+              "maxReceivesPerSecond": 1000
+            },
+            "requestPolicy": {
+              "headerContentType": "application/json"
+            },
+            "guaranteed": false
+          },
+          "Endpoint": "http://<host:1>/sns-endpoint",
+          "Owner": "111111111111",
+          "PendingConfirmation": "true",
+          "Protocol": "http",
+          "RawMessageDelivery": "false",
+          "SubscriptionArn": "arn:<partition>:sns:<region>:111111111111:<resource:1>:<resource:2>",
+          "SubscriptionPrincipal": "arn:<partition>:iam::111111111111:user/<resource:3>",
+          "TopicArn": "arn:<partition>:sns:<region>:111111111111:<resource:1>"
+        },
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "subscription-confirmation": {
+        "Message": "You have chosen to subscribe to the topic arn:<partition>:sns:<region>:111111111111:<resource:1>.\nTo confirm the subscription, visit the SubscribeURL included in this message.",
+        "MessageId": "<uuid:1>",
+        "Signature": "<signature>",
+        "SignatureVersion": "1",
+        "SigningCertURL": "<cert-domain>/SimpleNotificationService-<signing-cert-file:1>",
+        "SubscribeURL": "<subscribe-domain>/?Action=ConfirmSubscription&TopicArn=arn:<partition>:sns:<region>:111111111111:<resource:1>&Token=<token:1>",
+        "Timestamp": "date",
+        "Token": "<token:1>",
+        "TopicArn": "arn:<partition>:sns:<region>:111111111111:<resource:1>",
+        "Type": "SubscriptionConfirmation"
+      },
+      "http-confirm-sub-headers": {
+        "Accept-Encoding": "gzip,deflate",
+        "Connection": "connection",
+        "Content-Length": "content--length",
+        "Content-Type": "text/plain; charset=UTF-8",
```

Comment:
this shows that the `SubscriptionConfirmation` ignores the configured content-type